From 35d896d8afd30dc150e895bbbe3c186eebc9cb06 Mon Sep 17 00:00:00 2001
From: Thomas BOURGOIN <thomas.bourgoin@st.com>
Date: Mon, 16 Jan 2023 10:04:34 +0100
Subject: [PATCH] ARM: dts: stm32: add CRC support for STM32MP13x SoC family

This patch adds the CRC support for STM32MP13x SoC family.

Signed-off-by: Thomas BOURGOIN <thomas.bourgoin@st.com>
Change-Id: I4296506f3951baad5ea08eaf4d2a9bf286e41708
---
 arch/arm/boot/dts/st/stm32mp131.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/st/stm32mp131.dtsi b/arch/arm/boot/dts/st/stm32mp131.dtsi
index d4ad507cd1bb..ac3e902edf34 100644
--- a/arch/arm/boot/dts/st/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp131.dtsi
@@ -1448,6 +1448,13 @@ sdmmc2: mmc@58007000 {
 			status = "disabled";
 		};
 
+		crc1: crc@58009000 {
+			compatible = "st,stm32f7-crc";
+			reg = <0x58009000 0x400>;
+			clocks = <&rcc CRC1>;
+			status = "disabled";
+		};
+
 		eth1: eth1@5800a000 {
 			compatible = "snps,dwmac-4.20a", "st,stm32mp13-dwmac";
 			reg = <0x5800a000 0x2000>;
-- 
2.39.5

